app/src/main/res/layout/fragment_address_editor.xml

```diff
@@ -0,0 +1,371 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- This Source Code Form is subject to the terms of the Mozilla Public
+   - License, v. 2.0. If a copy of the MPL was not distributed with this
+   - file, You can obtain one at http://mozilla.org/MPL/2.0/. -->
+<androidx.core.widget.NestedScrollView
```

Comment:
This mostly reuses the existing [fragment_credit_card_editor.xml](https://searchfox.org/mozilla-mobile/source/fenix/app/src/main/res/layout/fragment_credit_card_editor.xml) to save us a bit of time and unblock work so that more people can work on this in parallel. I will file a follow up to Composify this in the future. I will note that this is not to spec with the proposed Figma design. When we were doing credit cards last year which is what the Address editor is also based on, we also deviated from the Font styles from the Figma spec since this is reusing the same styles that was used in Logins and Credit Cards. I will file a follow up if necessary to realign all our forms (Add search engine, Logins, Credit cards and Address) styles if UX determines that we want to adopt the new style for all of the forms, but otherwise, it seems okay to be consistent with the existing style everywhere for the time being.